src/_gettsim_tests/test_rounding.py

```diff
@@ -57,10 +63,15 @@ def test_func():
 
 
 @pytest.mark.parametrize(
-    "base, direction",
-    [(1, "upper"), ("0.1", "down"), (5, "closest")],
+    "base, direction, add",
+    [
+        (1, "upper", None),
```

Comment:
```suggestion
        (1, "upper", 0),
```
See above, I don't think this should work.